benchmark/nixlbench/src/worker/nixl/nixl_worker.cpp

```diff
@@ -620,10 +620,11 @@ xferBenchNixlWorker::exchangeIOV(const std::vector<std::vector<xferBenchIOV>> &l
                 buffer = (char *)calloc(desc_str_sz, sizeof(*buffer));
                 rt->recvChar((char *)buffer, desc_str_sz, srcrank);
 
-                std::string desc_str(buffer, desc_str_sz);
-                ser_des.importStr(desc_str);
+                const std::string desc_str(buffer, desc_str_sz);
+                nixlDeserializer des;
```

Comment:
minor: naming a bit inconsistent, you call serializer **n**ser, but deserializer just des